libs/ledger-live-common/src/config/sharedConfig.ts

```diff
@@ -0,0 +1,78 @@
+import { ConfigSchema } from "@ledgerhq/live-config/LiveConfig";
+
+const liveConfig: ConfigSchema = {
+  cosmos: {
+    type: "object",
+    default: {
+      axelar: {
+        lcd: "https://axelar-api.polkachu.com",
+        minGasPrice: 0.07,
+        ledgerValidator: "axelarvaloper1fgklp9hemczlwtqp9jqzq3xahh38hznx7vd805",
+      },
+      cosmos: {
+        lcd: "https://cosmoshub4.coin.ledger.com",
+        ledgerValidator: "cosmosvaloper10wljxpl03053h9690apmyeakly3ylhejrucvtm",
+        minGasPrice: 0.025,
+      },
+      desmos: {
+        lcd: "https://desmos-api.ibs.team",
+        minGasPrice: 0.0025,
+      },
+      dydx: {
+        lcd: "https://api.dydx.nodestake.top:443",
```

Comment:
is it me or there are extra spaces before some keys ? maybe run prettier

Reply:
it is weird. I don't find the "extra spaces" that you mentioned. 
by the way, this file is not longer used after my another PR https://github.com/LedgerHQ/ledger-live/pull/5763/commits/

I run the ```pnpm prettier ``` on my side and it seems that everything is OK